bin/crails

```diff
@@ -0,0 +1,3 @@
+#!/usr/bin/env bun
+
+require('../src/index.ts')
```

Comment:
For consistency you should use `import '../src/index.ts'` here too

Reply:
I didn't know that it was possible to import files like that!